helpers/mathjax.js

```diff
@@ -102,7 +103,15 @@ export function loadMathJax(mathJaxConfig) {
 				settings: { zoom: 'None' }
 			}
 		},
-		loader: { load: ['ui/menu'] },
+		loader: {
+			load: mathJaxConfig && mathJaxConfig.enableMML3Support
+				? [
+					'[mml]/mml3',
+					'ui/menu'
+				] : [
+					'ui/menu'
+				]
```

Comment:
I don't really know how MathJax will behave if some of these array elements are undefined or some default value. Given that this feature flag should be short lived (a few releases at most), I'm not really concerned about the boilerplate of reproducing this array.